modules/dnn/src/layers/einsum_layer.cpp

```diff
@@ -1389,6 +1293,112 @@ Mat LayerEinsumImpl::pairwiseOperandProcess(
     return output;
 };
 
+Mat LayerEinsumImpl::batchwiseMatMul(
```

Comment:
Are there any changes in this func? I propose to reduce number of changes and keep implementation in the top of the file (adding `LayerEinsumImpl::`)

Reply:
I checked difference. It's related to fastGemm only.